Constructor-io/constructorio-identity
# Extend storage options

- Remove `update_cookie` method
- More consistent method naming
- Add options to set naming for both client and session names (keys)
- Allow specification of where both client and session id's should be stored individually
- Backwards compatible
- 8 tests added
- Lint passes

---

**Manual testing:**
- Verified there is no direct access to cookie or local storage from our clients. There are a few spec files that make modifications, but nothing critical.
- Referenced local version of identity module and tested changing storage locations + setting different cookie + local storage key names

<img width="444" alt="Screen Shot 2020-06-06 at 6 12 22 PM" src="https://user-images.githubusercontent.com/4909263/83957046-59129a80-a821-11ea-820f-b6732002bf7f.png">

We should be good to go here! No change necessary on any of our clients, but we should change any client planning to use a backend integration to use cookies for both values rather than local storage.
